### ISSUE TITLE
feat(sequential): add sequential ordering options on a per-form basis

### ISSUE THREAD

**Issue by Benunc** [WordImpress/Give#3530]
## User Story

As a `give admin`, I want `to be able to add custom order ID prefixes on a per-form basis` so that `accounting folks can differentiate between forms based on ID`.

This would give me the ability to create a form for Fiscal year 2018, and another for 2019, and have the IDs demonstrate quickly how many donations have come in for each form. It also allows for third-party accounting software to quickly (by donation ID differentiate between forms/funds.

## Current Behavior
Currently the sequential ordering settings are global. 

## Expected Behavior
I expect to be able to set custom prefixes, etc on a per-form basis.

## Related
- customer request: https://secure.helpscout.net/conversation/629644509/22382/?folderId=848135 

## Tasks
- [ ] ensure that adding the functionality is opt-in in the same way that sequential ordering was to begin with (existing installs are not affected)
- [ ] when a new form is created, accept the global settings by default if sequential ordering is enabled

**Comment by kevinwhoffman** [comment]
From the customer:

> I'm setting up the plugin in the section for Sequential Ordering. Can we set it up, and reset it at the end of each year? For example... We start with FY18-0001, then next year start with FY19-0001.

The initial request seems more concerned with including the year in the sequential ID. That is a request I suggested when the feature was first in development, but we decided to keep it simple and reserve that advanced functionality for a potential add-on. I do think including a `{year}` placeholder in the prefix/suffix ix a valuable feature.

Regarding this issue, I do not see us adding the ability to customize sequential IDs on a per form basis. That would be a huge task and honestly I don't think that's what the customer wants in this case; they just want the year in the prefix. Closing.
